### PR TITLE
DEVEXP-518: E2E SMS/Groups

### PR DESCRIPTION
**Pull request by asein-sinch**
Notice: breaking change in the interface names:
 - `CreateGroupResponse` / `ReplaceGroupResponse` / `UpdateGroupResponse` / `GroupResponse` have all disappeared to become a single `Group` interface to reduce the confusion when using a response object from the SMS groups endpoints.

### CODE REVIEW

File: packages/sms/tests/rest/v1/groups/groups.steps.ts [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+      ],
+      add_from_group: '01W4FFL35P4NC4K35SMSGROUP2',
+      remove_from_group: '01W4FFL35P4NC4K35SMSGROUP3',
+    },
+  });
+});
+
+Then('the response contains the updated SMS group details', () => {
+  assert.equal(group.id, '01W4FFL35P4NC4K35SMSGROUP1');
+  assert.equal(group.name, undefined);
```
Comment (JPPortier) [inline packages/sms/tests/rest/v1/groups/groups.steps.ts:124]: This is exactly the purpose of the comment onto the feature PR's description
We need to ensure test is pointing this dedicated behavior: name field is missing not an empty string

Comment (asein-sinch) [reply packages/sms/tests/rest/v1/groups/groups.steps.ts]: Fixed

**Review by JPPortier — APPROVED** [review]
